src/main/java/edu/wpi/cs3733/D22/teamC/DBManager.java

```diff
@@ -146,4 +148,35 @@ public void initializeLocationTable() {
                 emptyTables
         );
     }
+    /**
+    Initialize Service Request Table
+     */
+    public void initializeSRTable() {
+        initializeTable("SERVICE_REQUESTS", "CREATE TABLE SERVICE_REQUESTS (\n" +
```

Comment:
Question about SQL convention: we use "LOCATION" singular for the Location's table and "SERVICE_REQUESTS" plural for this table. Which is preferred in SQL? We should standardize tables to follow a singular or plural convention early to avoid larger changes later on.